src/VisualStudio/CSharp/Impl/EventHookup/EventHookupSessionManager.cs

```diff
@@ -35,8 +44,28 @@ internal void EventHookupFoundInSession(EventHookupSession analyzedSession)
                 caretPoint.HasValue &&
                 analyzedSession.TrackingSpan.GetSpan(CurrentSession.TextView.TextSnapshot).Contains(caretPoint.Value))
             {
-                // Watch all text buffer changes & caret moves while this quick info session is
-                // active
+                Debug.Assert(_toolTipPresenter == null, "_toolTipPresenter is not null, we have an existing tooltip session active");
+
+                // Create a tooltip presenter that stays alive, even when the user types, without tracking the mouse.
+                _toolTipPresenter = this._toolTipService.CreatePresenter(analyzedSession.TextView,
+                    new ToolTipParameters(trackMouse: false, ignoreBufferChange: true));
+
+                // tooltips text is: Program_MyEvents;      (Press TAB to insert)
+                // GetEventNameTask() gets back the event name, only needs to add a semicolon after it.
+                var eventText = analyzedSession.GetEventNameTask.Result + ";";
+                var texts = new[] { eventText, CSharpEditorResources.Press_TAB_to_insert };
+                var textRuns = texts.Select(s => new ClassifiedTextRun(ClassificationTypeNames.Text, s));
+                var content = new[] { new ClassifiedTextElement(textRuns) };
+
+                // TODO: will file a bug for this
+                // use string[] for now to unblock tests for now, otherwise editor throws ArgumentNullException when running unit tests
```

Comment:
The exception that you are hitting is due to a missing ClassificationTypeDefinition for 'text'. You may have to adjust your MEF composition code to ensure that this is being pulled in for your tests.